kube-enforcer/templates/trivy-deployment.yaml

```diff
@@ -122,5 +131,5 @@ spec:
             privileged: false
             readOnlyRootFilesystem: true
       securityContext:
-        {}
+        { }
```

Comment:
@adishaull why did you added an empty string here?

Reply:
this is how we did the ke/starboard/envoy

Reply:
Before committing, i always do reformat code